js_modules/dagster-ui/packages/ui-core/src/runs/batchRunsForTimeline.tsx

```diff
@@ -32,8 +32,12 @@ export const batchRunsForTimeline = <R extends RunWithTime>(config: Config<R>) =
   const rangeLength = end - start;
 
   const now = Date.now();
-  const nowLeft = ((now - start) / (end - start)) * width;
 
+  // Give a pixel of breathing room for the "now" position.
+  const nowLeft = ((now - start) / (end - start)) * width + 1;
```

Comment:
Hmm, seems like you might be able to do this 1px shift in css?

Reply:
Ah to be clear, this is purely for the bucketing calculation.